wdl-grammar/src/tree.rs

```diff
@@ -479,6 +482,13 @@ impl SyntaxTree {
         &self.0
     }
 
+    /// Gets a copy of the underlying root green node for the tree.
+    pub fn green(&self) -> GreenNode {
```

Comment:
Minor nit: generally I've been returning a reference to the green node and letting the caller decide if it needs to be cloned or not.

Reply:
Fair enough. I introduced it as a convenience method for exactly that purpose but, to your point, the method is named more generally than that. I don't love any of the other method names, so I suppose I'll split the difference and require the caller to call `.into()`.